hugo/content/info/contrib.md

```diff
@@ -46,18 +41,19 @@ You should also seek consistency with the names of your volumes from prior years
 
 If your conference has satellite workshops, please make certain the workshops will actually be run.
 
+By default, the publication of papers associated with an event in the Anthology will be on the first day of the event (inclusive of workshops or tutorials).
+If you prefer to have it published on a different date, please inform us when you register.
+
 With this information in hand, we will assign you a [list of Anthology identifiers](https://docs.google.com/spreadsheets/d/166W-eIJX2rzCACbjpQYOaruJda7bTZrY7MBw_oa7B2E/edit?usp=sharing) for use with ACLPUB.
 
 ### Prepare your data
 
-Prepare your data (PDFs, BibTeX files, and metadata) using ACLPUB by following the instructions below.
-   You will run a script to lay it out into a single directory, and then send a link to your tarball.
-   **This tarball must be delivered to the Anthology director two weeks prior to your desired publication date**, which was negotiated when you first contacted us.
+Prepare your data (PDFs, BibTeX files, attachments, and metadata) by following the instructions below. You will end up with a tarball; this tarball (or a link to it) must be delivered to the Anthology director two weeks prior to your desired publication date**, which was negotiated when you first contacted us.
+
+For meetings using [Softconf](https://www.softconf.com)'s [STARTv2 conference management system](https://www.softconf.com/about/start-v2-mainmenu-26), data preparation is handled by the [ACLPUB](https://github.com/acl-org/ACLPUB/) package, whose output can be ingested into the Anthology.
+**Instructions for this can be found [here](https://github.com/acl-org/acl-pub)**.
 
```

Comment:
This is incorrect—the instructions are here: https://github.com/acl-org/ACLPUB/blob/master/anthologize/README.md.
They should probably be moved top-level in ACLPUB.

This is part of the reason [I want to rename acl-pub](https://github.com/acl-org/acl-pub/issues/17#issuecomment-554088102), to avoid this kind of confusion.

Reply:
The originally linked instructions are for the 2nd part of ACLPUB, which is run by conference-level pub chairs on their own computers, but the `acl-org` link contains instructions for both the 1st part, which runs on START, and the 2nd part. (The instructions over there should be updated to link to the `ACLPUB` repo instructions).

Reply:
Okay, updated.